src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -9,10 +11,14 @@ export interface BreadcrumbItem {
 const Breadcrumbs: React.FC<{ item: BreadcrumbItem }> = ({ item }) => {
   return (
     <nav aria-label="breadcrumbs">
-      <Style.BreadcrumbLink to={item.to}>
-        <Style.ArrowBackIcon fontSize="small" aria-hidden />
+      <TextButton
+        to={item.to}
+        startIcon={<ArrowBackIcon />}
+        trackingCategory={EventCategory.LEARN}
+        trackingLabel="Breadcrumbs"
+      >
         {item.label}
-      </Style.BreadcrumbLink>
+      </TextButton>
     </nav>
   );
```

Comment:
Maybe we should remove the `nav` wrapper, since this is not really a breadcrumbs element (changing it to `div` should work)